components/AppBar/examples/AppBarTypicalUseExample.m

```diff
@@ -39,6 +39,9 @@ - (id)init {
     // Optional: Change the App Bar's background color and tint color.
     UIColor *color = [UIColor colorWithWhite:0.2 alpha:1];
     _appBar.headerViewController.headerView.backgroundColor = color;
+    _appBar.headerViewController.headerView.shiftBehavior = MDCFlexibleHeaderShiftBehaviorEnabled;
+    [_appBar.headerViewController.headerView hideViewWhenShifted:_appBar.headerStackView];
```

Comment:
Should this be in the AppBar's logic?

Reply:
I thought about that, and it would be nice to make this the default, but I don't think we can as it will change the current behavior... WDYT?

Reply:
Ah true. I wager that we may want this to be the default behavior, so perhaps in a follow-up PR we can introduce an opt-in API where the content is faded out when enabled?

Reply:
Opt-in API on the AppBar would be nice. Thanks for preparing this PR so quickly!